Hudl.Mjolnir.Attributes/CommandAttribute.cs

```diff
@@ -15,10 +15,23 @@ public sealed class CommandAttribute : Attribute
         private readonly string _breakerKey;
         private readonly string _poolKey;
         private readonly int _timeout;
+        private readonly bool _ignoreTimeout;
 
         // See Mjolnir's Command constructors.
         public CommandAttribute(string group, string isolationKey, int timeout = DefaultTimeout) : this(group, isolationKey, isolationKey, timeout) { }
 
+        public CommandAttribute(string group, string isolationKey, bool ignoreTimeout)
```

Comment:
I don't think we do - ignoring timeouts seems like a conditional thing, so to have it baked into the attribute permanently feels odd.